internal/config/db.go

```diff
@@ -18,10 +19,31 @@ type Database struct {
 
 	QueryTimeout string `yaml:"query_timeout" json:"query_timeout" env:"QUERY_TIMEOUT" env-default:"5s"`
 
+	MySQL    MySQL    `yaml:"mysql" json:"mysql" env-prefix:"MYSQL_"`
 	SQLite   SQLite   `yaml:"sqlite" json:"sqlite" env-prefix:"SQLITE_"`
 	Postgres Postgres `yaml:"postgres" json:"postgres" env-prefix:"POSTGRES_"`
 }
 
+// MySQL defines configuration for using MySQL as the persistent store.
+type MySQL struct {
+	DBName      string           `yaml:"db_name" json:"db_name" env:"DB_NAME"`
+	Hostname    string           `yaml:"hostname" json:"hostname" env:"HOSTNAME"`
+	ReadReplica MySQLReadReplica `yaml:"read_replica" json:"read_replica" env-prefix:"READ_REPLICA_"`
```

Comment:
Does this make more sense as an array, since we may want to configure n number of read-replicas?

Reply:
Harry has specifically disallowed arrays: https://github.com/Kong/koko/blob/main/internal/config/types_test.go#L11

Additionally, I'm just copying what was done with Postgres:
https://github.com/Kong/koko/blob/main/internal/config/db.go#L30

I'd say if we'd want to expand this, we'd need to punt that off to another issue.